File: Plugins/Aspose_Cells_Java_for_Ruby/lib/asposecellsjava/splitpanes.rb
module Asposecellsjava
  module SplitPanes
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(__FILE__))) + '/data/'
        
        # Instantiating a Workbook object by excel file path
        workbook = Rjb::import('com.aspose.cells.Workbook').new(data_dir + 'Book1.xls')
        
        # Set the active cell
        workbook.getWorksheets().get(0).setActiveCell("A20")

        # Split the worksheet window
        workbook.getWorksheets().get(0).split()
        
        # Saving the modified Excel file in default (that is Excel 2003) format
        workbook.save(data_dir + "SplitPanes output.xls")

        puts "Panes split successfully."
    end
  end
end

File: Examples/src/main/java/com/aspose/cells/examples/worksheets/display/SplitPanes.java
package com.aspose.cells.examples.worksheets.display;

import com.aspose.cells.*;
import com.aspose.cells.examples.Utils;

public class SplitPanes {

    public static void main(String[] args) throws Exception {
        //ExStart:1
        // The path to the documents directory.
        String dataDir = Utils.getDataDir(SplitPanes.class);

        //Instantiate a new workbook
        //Open a template file
        Workbook book = new Workbook(dataDir + "book.xls");

        //Set the active cell
        book.getWorksheets().get(0).setActiveCell("A20");

        //Split the worksheet window
        book.getWorksheets().get(0).split();

        //Save the excel file
        book.save(dataDir + "book.out.xls", SaveFormat.EXCEL_97_TO_2003);

        //Print Message
        System.out.println("Panes split successfully.");
        //ExEnd:1
    }
}
